packages/next/taskfile.js

```diff
@@ -153,7 +153,19 @@ export async function copy_vercel_og(task, opts) {
           __dirname,
           dirname(require.resolve('@vercel/og/package.json'))
         ),
-        'dist/*.+(js|ttf|wasm),LICENSE'
+        '{./dist/*.+(js|ttf|wasm),LICENSE}'
+      )
+    )
+    .target('src/compiled/@vercel/og')
+
+  await task
+    .source(
+      join(
+        relative(
+          __dirname,
+          dirname(require.resolve('@vercel/og/package.json'))
+        ),
+        './dist/index.*.js'
```

Comment:
Why is this one needed? Wouldn't the regex above match?

Reply:
Example: 
<img width="402" alt="image" src="https://github.com/vercel/next.js/assets/229881/83463ba9-ea8b-490b-a75a-05c13aec8fb1">

Reply:
I feel it's issue of taskr itself, it didn't update the `index.node.js` and `index.edge.js` in the above regex. Have bumped to few limitation of regex with taskr

Reply:
@huozhi Does this work?

```
{./dist/*.+(js|ttf|wasm),./dist/index*.js,LICENSE}
```

Reply:
@styfle no luck 😢